lib/stitches/fill.py

```diff
@@ -85,7 +85,7 @@ def stitch_row(stitches, beg, end, angle, row_spacing, max_stitch_length, stagge
     offset = (first_stitch - beg).length()
 
     while offset < segment_length:
-        stitches.append(Stitch(beg + offset * row_direction, tags=('fill_row')))
```

Comment:
What, you don't like tags like "f", "i", "l", "l", "_", etc? ;)  (nice catch)

Reply:
Ha yes, it was a bit difficult to read ;)